src/main.rs

```diff
@@ -1031,6 +1031,8 @@ async fn run() -> Result<(), S2CliError> {
                     .bold()
                 );
 
+                total_data_len = 0;
```

Comment:
I think this will mess up the throughput calculation, although that is only helpful when catching up and not when tailing. Maybe remove the throughput & elapsed time stuff for now.